### PR TITLE
Update project.py

### PR DESCRIPTION
**Pull request by sirusbaladi**
added table visualizer + minor changes

### CODE REVIEW

File: code/project.py [hunk 1]
```diff
@@ -43,11 +43,32 @@ def query_db(sql: str):
     df = pd.DataFrame(data=data, columns=column_names)
 
     return df
+"## Read tables"
```
Comment (jbaranel) [inline code/project.py:46]: Why do we need this?

File: code/project.py [hunk 2]
```diff
@@ -59,12 +80,12 @@ def query_db(sql: str):
     st.write('Date range selected:', start_date, end_date)
 
 sql_sales_range = f"""
-    SELECT S.name as seller, SUM(P.price) as sum
```
Comment (jbaranel) [inline code/project.py:62]: Let's avoid refactoring things like this unless it is meaningful.

**Review by jbaranel — CHANGES_REQUESTED** [review]

File: code/project.py (hunk 2 above)
Comment (sirusbaladi) [reply code/project.py:62]: keep them like I edited

File: code/project.py (hunk 1 above)
Comment (sirusbaladi) [reply code/project.py:46]: nice to have, overall view of the data

File: code/project.py (hunk 2 above)
Comment (sirusbaladi) [reply code/project.py:62]: it's meaningful as it describes the data better than just "sum"

**Pull request merged**